jstests/sharding/read_pref_cmd.js

```diff
@@ -89,12 +89,11 @@ var testReadPreference = function(conn, hostList, isMongos, mode, tagSets, secEx
     /**
      * Assumption: all values are native types (no objects)
      */
-    var formatProfileQuery = function(queryObj, isEmbedded) {
-        var prefix = isEmbedded? 'command.query.' : 'command.';
+    var formatProfileQuery = function(queryObj) {
         var newObj = {};
 
         for (var field in queryObj) {
-            newObj[prefix + field] = queryObj[field];
+            newObj['command.' + field] = queryObj[field];
```

Comment:
note that these changes are because we now always unwrap a command before filling in a curOp. Although this is a slight behavior change, it is an improvement as the content of a profiling collection will be consistent irregardless of whether the node is a shard, a mongos, or a standalone.